make env error on macOS Mojave

I have Python 3.7 installed and pip3 19.0.1.

Running `make clean env` gets me these errors:
```
In --require-hashes mode, all requirements must have their versions pinned with ==. These do not:
    futures<4.0.0,>=2.2.0; python_version == "2.6" or python_version == "2.7" from https://files.pythonhosted.org/packages/2d/99/b2c4e9d5a30f6471e410a146232b4118e697fa3ffc06d6a65efde84debd0/futures-3.2.0-py2-none-any.whl#sha256=ec0a6cb848cc212002b9828c3e34c675e0c9ff6741dc445cab6fdd4e1085d1f1 (from s3transfer==0.1.13->-r requirements_base.txt (line 196))
    enum34; python_version < "3" from https://files.pythonhosted.org/packages/c5/db/e56e6b4bbac7c4a06de1c50de6fe1ef3810018ae11732a50f15f62c7d050/enum34-1.1.6-py2-none-any.whl#sha256=6bd0f6ad48ec2aa117d3d141940d484deccda84d4fcd884f5c3d93c23ecd8c79 (from cryptography==2.4.2->-r requirements_base.txt (line 220))
make: *** [env] Error 1
```
Does anyone have any ideas? I can run this fine in Ubuntu (in a VM on a different physical machine).

Possibly related: https://github.com/certbot/certbot/issues/5195

--- a/Makefile
+++ b/Makefile
@@ -77,7 +77,7 @@ pytest: env
 	$(py_test) --doctest-modules liberapay
 
 pytest-cov: env
-	PYTHONPATH=. $(py_test) --cov-report html --cov liberapay ./tests/py/
+	PYTHONPATH=. $(py_test) --cov-report html --cov liberapay ./tests/py/test_$${PYTEST-*}.py
 	@$(MAKE) --no-print-directory pyflakes
 	$(py_test) --doctest-modules liberapay
 

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-python := "$(shell { command -v python3.6 || command -v python; } 2>/dev/null)"
+python := "$(shell { command -v python3.6 || command -v python3.7; } 2>/dev/null)"
 install_where := $(shell $(python) -c "import sys; print('' if hasattr(sys, 'real_prefix') else '--user')")
 
 # Set the relative path to installed binaries under the project virtualenv.

--- a/Makefile
+++ b/Makefile
@@ -57,6 +57,11 @@ run: env
 py: env
 	PYTHONPATH=. $(with_local_env) $(env_py) -i $${main-liberapay/main.py}
 
+shell: py
+
+test-shell: env
+	PYTHONPATH=. $(with_tests_env) $(env_py) -i $${main-liberapay/main.py}
+
 test-schema: env
 	$(with_tests_env) ./recreate-schema.sh test
 

